dts/msm8916/msm8916-mtp.dts

```diff
@@ -39,4 +39,33 @@
 		compatible = "asus,z010d", "qcom,msm8916", "lk2nd,device";
 		lk2nd,match-cmdline = "* mdss_mdp.panel=1:dsi:0:qcom,mdss_dsi_r69339_720p_video:1:none";
 	};
+
+	asus-p024 {
+		model = "ASUS ZenPad 8.0";
+		compatible = "asus,p024", "qcom,msm8916", "lk2nd,device";
+		lk2nd,match-panel;
+
+		panel {
+			compatible = "asus,p024-panel";
+
+			qcom,mdss_dsi_ili6136s_800x1280_video {
+				compatible = "asus,ili6136s_800x1280";
```

Comment:
Don't use underscores (`_`) in compatible names please. I'd suggest something like

- `asus,p024-ili6136s`
- `asus,p024-nt35521`
- ...
- `asus,p024-rm68200-kd`